app/assets/stylesheets/not-logged-in/_not-logged-in-wrapper.css

```
@@ -7,7 +7,7 @@
   content: "";
   background-image: url(https://bootcamp.fjord.jp/images/background/background.svg);
   background-repeat: repeat;
-  position: fixed;
+  background-repeat-position: fixed;
```

Biome (2.3.13)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)